src/test/java/org/opensearch/security/ccstest/CrossClusterSearchTests.java

```diff
@@ -84,7 +84,9 @@ public ClusterTransportClientSettings() {
         }
 
         public ClusterTransportClientSettings(Settings clusterSettings, Settings transportSettings) {
-            super(clusterSettings, transportSettings);
+            super(Settings.builder()
+                .put(clusterSettings)
+                .putList("node.roles", "remote_cluster_client").build(), transportSettings);
```

Comment:
This is required for cross-cluster to be enabled https://opensearch.org/docs/latest/replication-plugin/get-started/#prerequisites

Note; I did not investigate why this wasn't required before.